spec/features/admin/product_import_spec.rb

```diff
@@ -392,6 +395,14 @@
 
       expect(page).to have_selector '.created-count', text: '2'
       expect(page).to have_no_selector '.updated-count'
+
+      visit spree.admin_products_path
+
+      within "#p_#{Spree::Product.find_by(name: 'Carrots').id}" do
+        expect(page).to have_input "product_name", with: "Carrots"
+        expect(page).to have_select "variant_unit_with_scale", selected: "Weight (lb)"
+        expect(page).to have_content "5" #on_hand
```

Comment:
This would also match `500`. You could add a more specific selector but it's probably fine here.